### PR TITLE
grammaradd.html

### PR DESCRIPTION
**Pull request by komalthesomething**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ Contributions are welcome !
...
-If you would like to add a new package consult
+If you would like to add a new package, kindly consult
 [these instructions](https://opam.ocaml.org/doc/Packaging.html#Publishing).
 
 ## License
 
 All the metadata contained in this repository are licensed under the
 [CC0 1.0 Universal](http://creativecommons.org/publicdomain/zero/1.0/)
-license.
+license respectfully.
```
Comment (kit-ty-kate) [inline README.md]: I'm sorry but this change doesn't make sense, I think?

**Comment by mseri** [comment]
I don't see the point of this PR to be honest

**Comment by kit-ty-kate** [comment]
me neither

File: README.md (hunk 1 above)
Comment (pmetzger) [reply README.md]: It doesn't make sense.

**Comment by pmetzger** [comment]
I think this should be closed.

**Pull request closed without merging**